documentation/general/dash-sonic-hld.md

```diff
@@ -310,11 +320,13 @@ DASH_ACL_RULE_TABLE:{{group_id}}:{{rule_num}}
     "priority": {{priority}}
     "action": {{action}}
     "terminating": {{bool}}
-    "protocol": {{list of protocols}}
-    "src_addr": {{list of address}}
-    "dst_addr": {{list of address}}
-    "src_port": {{list of range of ports}}
-    "dst_port": {{list of range of ports}}
+    "protocol": {{list of protocols}} (OPTIONAL)
+    "src_tag": {{list of tag name}} (OPTIONAL)
```

Comment:
1. `specifying a src_tag with empty prefix must not match packets.` I add this description under the TAG table.
2. `When specifying a src_tag and src_prefix, it shall be a OR. Please clarify it in documentation`. I think this should be an AND instead of OR. If you want a OR, just to define another rule. 
There are three reasons:
1. The tag fields are same as other fields like protocol, port and so on. The relationship between src, protocol and port should be AND.
2. The TAG is for identifying a packets that belongs to a specific group. In the further, maybe we can identify a packets not only from address but also from other features.
3. I think if you want to implement `OR` in P4, we have to separate the TAG and ADDR to different ACL tables, which will increase the complexity of the ACL system.

If you accept my points, I can polish and clarify the descriptions for TAG.